src/OpenAL/Silk.NET.OpenAL/AL/AL.cs

```diff
@@ -358,7 +358,8 @@ public static AL GetApi(bool soft = false)
         public TExtension GetExtension<TExtension>()
             where TExtension : NativeExtension<AL>
         {
-            return IsExtensionPresent(ExtensionAttribute.GetExtensionAttribute(typeof(TExtension)).Name)
+            var extAttr = ExtensionAttribute.GetExtensionAttribute(typeof(TExtension));
+            return (extAttr is null || IsExtensionPresent(extAttr.Name))
```

Comment:
that check doesn't look right? Do we want to create extension types without an attribute?

Reply:
There was a reason why I thought it was necessary, but I've since forgotten it. Reason #5435 for commenting. Removed!